doc/releases/changelog-0.24.0.md

```diff
@@ -585,6 +585,9 @@ When `decomposition()` is called, a list of new operators equal to this one rais
   submodule. The developer-facing `ControlledOperation` class is no longer imported top-level.
   [(#2656)](https://github.com/PennyLaneAI/pennylane/pull/2656)
 
+* Quantum chemistry operations are now remarkably faster with new highly-performant kernels added to `lighting.qubit` and 
+  `lightning.gpu`. For more details, please see the release notes for [lightning.qubit](https://github.com/PennyLaneAI/pennylane-lightning/releases/tag/v0.24.0) and [lightning.gpu](https://github.com/PennyLaneAI/pennylane-lightning-gpu/releases/tag/v0.24.0). 
+* 
```

Comment:
@mlxd can we get PR hyperlinks to these additions?

Reply:
Does it make sense to do that here? I think linking the Lightning PR release log is probably fine, as the merges are in that repo rather than here. What do you think?

Reply:
Yes it was a misunderstanding, it has been removed

Reply:
Yep sorry about that! n00b here 🤪

Reply:
No worries at all :)